workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+#endif
+
+        public void SetConnection(Connection connection)
+        {
+            this.connection = connection;
+        }
+
+        public void HandleLog(LogType type, LogEvent logEvent)
+        {
+#if UNITY_EDITOR
+            if (Application.isEditor)
+            {
+                loggingDispatcher.HandleLog(type, logEvent);
```

Comment:
I think we should always log to Unity as well. If they want to disable it, they can change the setting as described here: https://answers.unity.com/questions/1301347/how-to-disable-all-logging-on-release-build.html

Reply:
Or they just change the dispatcher. That is what it is there for.